src/test/java/codeu/controller/ChatServletTest.java

```diff
@@ -47,6 +47,10 @@
   private MessageStore mockMessageStore;
   private UserStore mockUserStore;
 
+  private final String TEST_USERNAME = "test username";
```

Comment:
`static final String` (same for all)
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

I might put this at module level, too, but it's probably okay here too